plugins/docusaurus-plugin-ionic-component-api/index.js

```diff
@@ -132,14 +132,31 @@ ${properties
   .map((prop) => {
     const isDeprecated = prop.deprecation !== undefined;
 
+    /**
+     * Several components link to /docs/theming/basics
+     * in the generated core.json file from @ionic/core.
+     * This script updates those references to point to
+     * the new URL.
+     * The docs for v5 of `ion-select` and `ion-datetime`
+     * have incorrect links when linking to other
+     * components. This is also adjusted when processing the
+     * properties description.
+     */
     const docs = isDeprecated ? `${prop.docs}\n_Deprecated_ ${prop.deprecation}` : prop.docs;
+    const description = formatMultiline(docs)
+    const formattedDescription = description
+      .replace('/docs/theming/basics', '/docs/guide/theming/basics')
+      .replace('../alert', 'alert')
+      .replace('../action-sheet', 'action-sheet')
+      .replace('../popover', 'popover')
+      .replace('../picker', 'picker');
```

Comment:
Would it be possible to fix this directly in the framework repo instead? Having this code hang around seems like it could cause hard-to-troubleshoot bugs down the road. Ditto for the `list` fix below.

Reply:
Yeah I guess what we could do is do a v5 patch that fixes the issue 🤔 I can schedule some time for that